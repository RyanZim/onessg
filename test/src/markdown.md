Heading
=======

Sub-Heading
-----------

### Sub-sub-heading

Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor
incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis
nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.

Duis **aute** irure dolor in _reprehenderit_ in voluptate velit esse cillum dolore eu fugiat nulla pariatur. *Excepteur* sint occaecat cupidatat non __proident__, sunt in culpa qui officia deserunt mollit anim id est laborum.

A [link](http://example.com/).

## Lists

- Unordered
- List

---

1. One
2. Two
3. Three

## Table

| Header One     | Header Two     |
| :------------- | :------------- |
| Item One       | Item Two       |
