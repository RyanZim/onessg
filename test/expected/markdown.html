<h1 id="heading">Heading</h1>
<h2 id="sub-heading">Sub-Heading</h2>
<h3 id="sub-sub-heading">Sub-sub-heading</h3>
<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor
incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis
nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
<p>Duis <strong>aute</strong> irure dolor in <em>reprehenderit</em> in voluptate velit esse cillum dolore eu fugiat nulla pariatur. <em>Excepteur</em> sint occaecat cupidatat non <strong>proident</strong>, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
<p>A <a href="http://example.com/">link</a>.</p>
<h2 id="lists">Lists</h2>
<ul>
<li>Unordered</li>
<li>List</li>
</ul>
<hr>
<ol>
<li>One</li>
<li>Two</li>
<li>Three</li>
</ol>
<h2 id="table">Table</h2>
<table>
<thead>
<tr>
<th style="text-align:left">Header One</th>
<th style="text-align:left">Header Two</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left">Item One</td>
<td style="text-align:left">Item Two</td>
</tr>
</tbody>
</table>
